src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectResource.java

```diff
@@ -196,6 +183,35 @@ public Response showBranchOverview(@Context HttpServletRequest request,
 			} else throw e;
 		}
 
+		switch (branchDeletion) {
```

Comment:
Anyone got any tips on how to do this? This is the only way I can generalize the generating of the branch overview, while also being able to add a feedback popup after branch deletion.